### PR TITLE
Community Contribution: Minor README tweaks.

### PR DESCRIPTION
**Pull request by oubiwann**
Some minor fixes to the README from @tmm2018.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,20 +17,16 @@ Search API provides access to this metadata.
...
   - Access Control Lists (ACLs) are the mechanism by which users are granted
-    access to perform different operations in the CMR. CMR ACLs follow the
-    same design as ECHO ACLs which in turn are derived from the generic ACL
+    access to perform different operations in the CMR. CMR ACLs follow the same design as ECHO ACLs which in turn are derived from the generic ACL
     design pattern used in many other systems. At a high level, an ACL is a
     mapping of actors (subjects) to resources (object) to operations
-    (predicate). For instance, a CMR ACL might specify that all Registered
-    users have READ access to ASTER data or all users in a provider operations
-    group have permissions to ingest data for a particular provider.
+    (predicate). For instance, a CMR ACL might specify that all Registered users have READ access to ASTER data or all users in a provider operations group has permissions to ingest data for a particular provider.
```
Comment (oubiwann) [inline README.md]: "have" was correct; the change to "has" is not grammatically correct ("all users ... have").

File: README.md [hunk 2]
```diff
@@ -85,8 +79,8 @@ services as well as starting up worker threads.
 
 #### Building and Running separate CMR Applications
 
-This will build all of the applications, but will put each jar into the
-appropriate /target directory for each application. The command shown in step
+This will build all of the applications but will put each jar into the
+appropriate /target directory for each application. The command is shown in step
```
Comment (oubiwann) [inline README.md]: The addition of "is" here is incorrect (it adds two verbs to the sentence).

**Comment by tmm2018** [comment]
thank you so much for taking the time to fix this. gently. T.

**Comment by oubiwann** [comment]
Okay, I've pushed the tiny fixes and this should be ready for further review!

**Comment by oubiwann** [comment]
Thanks guys, and thanks for your contribution @tmm2018!

**Pull request merged**